src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -154,11 +160,37 @@ def image_refs(
         )
 
 
-def get_build_options(target: Target) -> Iterator[str]:
+def get_build_options(
+    build_target_stage: str | None,
```

Comment:
I know this is the name of the option, but it's a little confusing here. Maybe `global_target_stage_option`?